### PR TITLE
Upgrade to Prettier 1.19

### PR DESCRIPTION
**Pull request by j-f1**
This updates Prettier to the latest `master`, which will eventually become 1.19. This is mostly to see what changes occur and make sure there aren’t any regressions. I plan to update this PR to use the regular stable version of 1.19 once it is released.

### CODE REVIEW

File: app/test/helpers/repositories.ts [hunk 1]
```diff
@@ -185,7 +185,10 @@ export async function setupConflictedRepoWithMultipleFiles(): Promise<
   const repo = await setupEmptyRepository()
 
   const firstCommit = {
-    entries: [{ path: 'foo', contents: 'b0' }, { path: 'bar', contents: 'b0' }],
+    entries: [
+      { path: 'foo', contents: 'b0' },
+      { path: 'bar', contents: 'b0' },
+    ],
```
Comment (lydell) [inline app/test/helpers/repositories.ts]: :+1:

File: app/test/unit/git/diff-check-test.ts [hunk 2]
```diff
@@ -28,7 +28,11 @@ describe('getFilesWithConflictMarkers', () => {
     })
     it('finds multiple conflicted files', async () => {
       expect(await getFilesWithConflictMarkers(repository.path)).toEqual(
-        new Map([['baz', 3], ['cat', 3], ['foo', 3]])
+        new Map([
+          ['baz', 3],
+          ['cat', 3],
+          ['foo', 3],
+        ])
```
Comment (lydell) [inline app/test/unit/git/diff-check-test.ts]: :+1:

File: app/src/lib/api.ts [hunk 3]
```diff
@@ -577,9 +577,7 @@ export class API {
       if (e instanceof APIError) {
```
Comment (rexingQ) [inline app/src/lib/api.ts]:
```suggestion
      if (e instanceof APIError) {
```
```suggestion
      if (e instanceof APIError) {
```

**Comment by outofambit** [comment]
:wave: hi @j-f1! i know it's been awhile for this PR and i'd like to get it in. would you like to merge in the latest `development` and get this passing CI? if not, no worries, I can pick up where you left off here. thanks!

**Comment by j-f1** [comment]
@outofambit the changes that need human intervention to upgrade are minimal, so I think it would be best to close this PR since it has a bunch of conflicts. I have finals right now so I don’t really have the capacity to do a prettier 2.0 upgrade right now.

**Comment by Rexogamer** [comment]
(sorry if off topic but) good luck with your finals!

**Pull request closed without merging**